### DISCUSSION TITLE
trough ball count

### DISCUSSION THREAD

**Discussion by Lieven-Funsystems** [missionpinball/missionpinball#33] [Q&A]
Hi i want to let the player now that there have to be enough balls in the  through to start a play.
balls_installed: 3
min_balls: 3
 
mode_attract_started{ball_device.bd_trough.balls!=3}: add_balls   does not work
i tried with available_balls==3,  ball_count==3 but no avail
what is the right syntax to know how many balls there are in the trough?

thanks,
Lieven

**Comment by JAMESCARDONA-PINBALL** [comment]
if you set min_balls: to 3 as you did above then a game will not start if there are not enough balls in the trough. What I do is post a screen message when the player presses start if there are not enough balls. A better solution would be to run ball search manually as well. Please note that in MPF ball search is enabled during attract mode so this should run automatically.

**Reply by Lieven-Funsystems** [reply]
Thats what i try to accomplish. a message when there are less than 3 balls in the through but the condition does not work.

**Reply by Lieven-Funsystems** [reply]
And ball search is working as it should, its just to counter the fact that there are less than 3 balls in the pinball installed
